heron/executor/src/python/heron_executor.py

```diff
@@ -677,7 +677,7 @@ def _untar_if_needed(self):
     if self.pkg_type == "tar":
       os.system("tar -xvf %s" % self.topology_bin_file)
     elif self.pkg_type == "pex":
-      os.system("unzip %s" % self.topology_bin_file)
+      os.system("unzip -qq -n %s" % self.topology_bin_file)
```

Comment:
What happens when you relaunch the topology after changing code? Then the new code will not take effect?